from PyQt5 import QtCore, QtGui, QtWidgets

class Ui_FridayGUI(object):
    def setupUi(self, FridayGUI):
        FridayGUI.setObjectName("FridayGUI")
        FridayGUI.resize(940, 615)
        self.centralwidget = QtWidgets.QWidget(FridayGUI)
        self.centralwidget.setObjectName("centralwidget")
        self.label = QtWidgets.QLabel(self.centralwidget)
        self.label.setGeometry(QtCore.QRect(0, 0, 941, 611))
        font = QtGui.QFont()
        font.setFamily("Cascadia Code Light")
        font.setPointSize(12)
        self.label.setFont(font)
        self.label.setText("")
        self.label.setPixmap(QtGui.QPixmap("GUI Resources/BG.gif"))
        self.label.setScaledContents(True)
        self.label.setObjectName("label")
        self.label_2 = QtWidgets.QLabel(self.centralwidget)
        self.label_2.setGeometry(QtCore.QRect(0, -20, 321, 161))
        self.label_2.setText("")
        self.label_2.setPixmap(QtGui.QPixmap("GUI Resources/Initializing.gif"))
        self.label_2.setScaledContents(True)
        self.label_2.setObjectName("label_2")
        self.label_3 = QtWidgets.QLabel(self.centralwidget)
        self.label_3.setGeometry(QtCore.QRect(10, 420, 191, 171))
        self.label_3.setText("")
        self.label_3.setPixmap(QtGui.QPixmap("GUI Resources/F.R.I.D.A.Y-modified.png"))
        self.label_3.setScaledContents(True)
        self.label_3.setObjectName("label_3")
        self.label_4 = QtWidgets.QLabel(self.centralwidget)
        self.label_4.setGeometry(QtCore.QRect(20, 170, 101, 181))
        self.label_4.setText("")
        self.label_4.setPixmap(QtGui.QPixmap("GUI Resources/LoadingBoxHorizontal.PNG"))
        self.label_4.setScaledContents(True)
        self.label_4.setObjectName("label_4")
        self.label_5 = QtWidgets.QLabel(self.centralwidget)
        self.label_5.setGeometry(QtCore.QRect(800, 250, 121, 161))
        self.label_5.setText("")
        self.label_5.setPixmap(QtGui.QPixmap("GUI Resources/LoadingBoxVertical.PNG"))
        self.label_5.setScaledContents(True)
        self.label_5.setObjectName("label_5")
        self.label_6 = QtWidgets.QLabel(self.centralwidget)
        self.label_6.setGeometry(QtCore.QRect(710, 10, 221, 201))
        self.label_6.setText("")
        self.label_6.setPixmap(QtGui.QPixmap("GUI Resources/Compass.gif"))
        self.label_6.setScaledContents(True)
        self.label_6.setObjectName("label_6")
        self.label_7 = QtWidgets.QLabel(self.centralwidget)
        self.label_7.setGeometry(QtCore.QRect(760, 450, 181, 131))
        self.label_7.setText("")
        self.label_7.setPixmap(QtGui.QPixmap("GUI Resources/Loading.gif"))
        self.label_7.setScaledContents(True)
        self.label_7.setObjectName("label_7")
        self.label_8 = QtWidgets.QLabel(self.centralwidget)
        self.label_8.setGeometry(QtCore.QRect(220, 530, 521, 61))
        self.label_8.setText("")
        self.label_8.setPixmap(QtGui.QPixmap("GUI Resources/StarkIndustries.jpg"))
        self.label_8.setScaledContents(True)
        self.label_8.setObjectName("label_8")
        self.pushButton = QtWidgets.QPushButton(self.centralwidget)
        self.pushButton.setGeometry(QtCore.QRect(660, 450, 91, 31))
        font = QtGui.QFont()
        font.setPointSize(11)
        font.setBold(True)
        font.setWeight(75)
        self.pushButton.setFont(font)
        self.pushButton.setAutoFillBackground(False)
        self.pushButton.setStyleSheet("background-color: rgb(255, 255, 0);")
        self.pushButton.setObjectName("pushButton")
        self.pushButton_2 = QtWidgets.QPushButton(self.centralwidget)
        self.pushButton_2.setGeometry(QtCore.QRect(660, 480, 91, 31))
        font = QtGui.QFont()
        font.setPointSize(11)
        font.setBold(True)
        font.setWeight(75)
        self.pushButton_2.setFont(font)
        self.pushButton_2.setAutoFillBackground(False)
        self.pushButton_2.setStyleSheet("background-color: rgb(0, 170, 0);")
        self.pushButton_2.setObjectName("pushButton_2")
        self.textBrowser = QtWidgets.QTextBrowser(self.centralwidget)
        self.textBrowser.setGeometry(QtCore.QRect(0, 90, 111, 31))
        self.textBrowser.setStyleSheet("background:transparent;\n"
                                         "border-radius:none;\n"
                                         "font-size:18px;\n"
                                         "color:white;")
        self.textBrowser.setObjectName("textBrowser")
        self.textBrowser_2 = QtWidgets.QTextBrowser(self.centralwidget)
        self.textBrowser_2.setGeometry(QtCore.QRect(140, 80, 131, 31))
        self.textBrowser_2.setStyleSheet("background:transparent;\n"
                                         "border-radius:none;\n"
                                         "font-size:18px;\n"
                                         "color:white;")
        self.textBrowser_2.setObjectName("textBrowser_2")
        FridayGUI.setCentralWidget(self.centralwidget)

        self.retranslateUi(FridayGUI)
        QtCore.QMetaObject.connectSlotsByName(FridayGUI)

    def retranslateUi(self, FridayGUI):
        _translate = QtCore.QCoreApplication.translate
        FridayGUI.setWindowTitle(_translate("FridayGUI", "MainWindow"))
        self.pushButton.setText(_translate("FridayGUI", "TRAIN"))
        self.pushButton_2.setText(_translate("FridayGUI", "START"))


if __name__ == "__main__":
    import sys
    app = QtWidgets.QApplication(sys.argv)
    FridayGUI = QtWidgets.QMainWindow()
    ui = Ui_FridayGUI()
    ui.setupUi(FridayGUI)
    FridayGUI.show()
    sys.exit(app.exec_())
